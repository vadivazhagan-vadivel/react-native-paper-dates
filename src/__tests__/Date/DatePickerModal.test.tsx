import * as React from 'react'
import { SafeAreaProvider } from 'react-native-safe-area-context'

import renderer from 'react-test-renderer'
import DatePickerModal from '../../Date/DatePickerModal'

it('renders DatePickerModal', () => {
  const tree = renderer
    .create(
      <SafeAreaProvider>
        <DatePickerModal
          locale="en"
          mode="single"
          visible
          onDismiss={() => null}
          date={new Date('01/01/2022')}
          onConfirm={() => null}
          themeValue={{
            secondaryColor: '#fff',
            primaryColor: '#1060e0',
            accentColor: '#070707',
            backgroundColor: '#ECE9E1',
            fontFamily: 'Roboto',
          }}
        />
      </SafeAreaProvider>
    )
    .toJSON()

  expect(tree).toMatchSnapshot()
})
