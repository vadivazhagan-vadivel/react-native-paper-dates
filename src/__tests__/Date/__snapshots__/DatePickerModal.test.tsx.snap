// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders DatePickerModal 1`] = `
<RNCSafeAreaProvider
  onInsetsChange={[Function]}
  style={
    [
      {
        "flex": 1,
      },
      undefined,
    ]
  }
/>
`;
